plugins/pulp_rpm/plugins/importers/yum/listener.py

```diff
@@ -130,8 +135,9 @@ def _verify_size(self, model, report):
 
     def _verify_checksum(self, model, report):
         """
-        Verifies the checksum of the given unit if the sync is configured to do so. If the verification
-        fails, the error is noted in this instance's progress report and the error is re-raised.
+        Verifies the checksum of the given unit if the sync is configured to do so. If the
+        verification fails, the error is noted in this instance's progress report and the error
+        is re-raised.
```

Comment:
This change too. It's a good change, but it should be in a separate pull request.